portfolio/src/main/webapp/budget.js

```diff
@@ -0,0 +1,160 @@
+function isNumber(evt){
+    var iKeyCode = (evt.which) ? evt.which : evt.keycode
+    //Only return if the ASCII code of the input is between 48 and 57 (ASCII for number)
```

Comment:
It would be nice to know what 48 and 57 are for keypresses (0 and 9, respectively?)